basicrta/contacts.py

```diff
@@ -193,7 +193,7 @@ def run(self):
         # os.remove('.tmpmap')
         # cfiles = glob.glob('.contacts*')
         # [os.remove(f) for f in cfiles]
-        print(f'\nSaved contacts to "contacts_{self.cutoff}.npy"')
+        print(f'\nSaved contacts to "contacts_{self.cutoff}.pkl"')
```

Comment:
Make sure that the  existing docs reflect the use of the pkl suffix for the contacts file.

Reply:
docs/source/tutorial.rst contains an instance of contacts.npy. Please fix to read contacts.pkl for consistency.